torch/utils/data/dataset.py

```diff
@@ -87,20 +90,15 @@ def register_function(cls, function_name, function):
         cls.functions[function_name] = function
 
     @classmethod
-    def register_datapipe_as_function(cls, function_name, cls_to_register, enable_df_api_tracing=False):
+    def register_datapipe_as_function(cls, function_name, cls_to_register):
```

Comment:
My understanding is that `enable_df_api_tracing` is removed because the conversion between `DataPipe` and `DataFrame` should only work for `IterDataPipe` (particularly `DFIterDataPipe`), correct?

Reply:
Yep. It can be added when we have better understanding about `DFMapDataPipe` (really low priority for now).